assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -98,7 +99,10 @@
 ]
 
 
-FORCE_4BIT_QUANTIZATION_MODELS = ["Llama-2-70b", "tiiuae-falcon-40b"]
+FORCE_4BIT_QUANTIZATION_MODELS = [
```

Comment:
I believe, we can remove this check?
cc @amangupta26

Reply:
We are not forcing Quantization anymore from script